lib/notifier.js

```diff
@@ -2,20 +2,22 @@ const notifier = require('node-notifier');
 const { config } = require('./config');
 const telegramBot = require('./telegram-bot');
 const cache = {};
+const util = require('util')
 
 module.exports = {
     notifyIfChanged
 };
 
 function notifyIfChanged(businesses) {
-    const message = createMessage(businesses);
     const options = config.get('notifications');
+    var message = createMessage(businesses,options);
+    message = (!message)?'<i>no good TGTG grabs available right now</i>':message;
     if(options.console.enabled){
-        notifyConsole(message, options.console);
+        notifyConsole(message.replace( /(<([^>]+)>)/ig, ''), options.console);
```

Comment:
Stripping out the HTML is of course an ugly approach. There are 2 ways to solve this properly:
1. Pass the businisses to `telegramBot.notify` so it can create its own message.
2. Create a separate `createHtmlMessage` function.